### ISSUE TITLE
Add a user dashboard

### ISSUE THREAD

**Issue by ExpressHermes** [ExpressHermes/Helping-Hand#11]
Create a seperate view to show user dashboard

**Comment by Halix267** [comment]
@ExpressHermes What information should be on dashboard?

**Comment by ExpressHermes** [comment]
See the  user model in `models.py`. If you have any suggestions regarding additional info that we can have for users, hard code them in front end. I will create an issue to make a view to add such additional information.

**Comment by praneeth-rdy** [comment]
Maybe we can create a new model with name **Profile**. One of its field can be **OneToOneField** which stores user object to which this Profile belongs to and the other fields can contain additional info of user like events created by them, names of users who are interested in their events, etc

Any opinions on this?

**Comment by ExpressHermes** [comment]
@Praneeth-rdy It's a good idea. Go ahead and implement it.

**Comment by praneeth-rdy** [comment]
Yeah! Working on it. I created a **CustomUser** model instead of a separate **Profile** model. Backend work is done. Working on frontend.

**Comment by erbaibhav** [comment]
@ExpressHermes What you want to update for user dashboard ? PLease elaborate so that i can take this task!

**Comment by ExpressHermes** [comment]
Hey @baibhav3211 PR #17 is a good reference of what is needed. You can look in this PR for details!

**Comment by erbaibhav** [comment]
@ExpressHermes I understood what to create...but i have some doubts! How to discuss?

**Comment by ExpressHermes** [comment]
join slack for the project here: [invite link](https://join.slack.com/t/expresshermes-kwoc21/shared_invite/zt-zrefahmo-NcRHial9jkoZrzZEcFMMTA)

You can DM me there.

**Comment by erbaibhav** [comment]
@ExpressHermes PLease asssign this task to me! Thanks :)
